control/control.yaml

```diff
@@ -65,50 +65,31 @@ components:
           type: array
           items:
             type: string
-            x-constraint:
-            - '/components/schemas/Port/properties/name'
+          x-constraint:
+          - '/components/schemas/Port/properties/name'
         state:
           description: >-
             The capture state.
           type: string
           enum: [start, stop]
 
-    Bgpv4.Route.State:
+    Route.State:
       description: >-
-        BGPv4 route specific action
+        Sets the device route state
       type: object
       required: [state]
       properties:
         names:
           description: >-
-            The names of Bgpv4RouteRange object to control.
+            The names of device route objects to control.
+            If no names are specified then all route objects will be affected.
           type: array
           items:
             type: string
-            x-constraint:
-            - '/components/schemas/Device.Bgpv4RouteRange/properties/name'
+          x-constraint:
+          - '/components/schemas/Device.Bgp.Route/properties/name'
```

Comment:
`Device.Bgp.Route` node is not available, because of which `set_route_state`  is not getting generated in snappi

Reply:
aab67e5